agents/architect.md

```
@@ -0,0 +1,119 @@
+  ```
+- **Error Cases**:
+  - 400: [조건]
+  - 403: [조건]
+  - 404: [조건]
+
+---
+
+### DB 스키마 변경
+
+#### 신규 테이블: `table_name`
+| 컬럼명 | 타입 | 제약조건 | 설명 |
```

markdownlint-cli2 (0.21.0)

[warning] Tables should be surrounded by blank lines

(MD058, blanks-around-tables)